conan/tools/google/bazeldeps.py

```diff
@@ -118,15 +118,10 @@ def _save_lib_path(lib_, lib_path_):
             full_path = os.path.join(libdir, f)
             if not os.path.isfile(full_path):  # Make sure that directories are excluded
                 continue
-            # Users may not name their libraries in a conventional way. For example, directly
```

Comment:
But this seems it was solving an issue in the past, of libraries without a conventional extensions?

Reply:
You were right. Original issue: https://github.com/conan-io/conan/issues/11331
Covered and fixed, thanks!